src/metaswitch/clearwater/cluster_manager/plugin_utils.py

```diff
@@ -156,8 +156,8 @@ def leave_cassandra_cluster():
     except:
         start_cassandra()
 
+    run_command("monit unmonitor cassandra")
```

Comment:
I wasn't convinced that this should be in the plugin, but it's probably right - this means the orchestrator doesn't have to care about whether a node has Cassandra or not, it just calls `service clearwater-etcd decommission` everywhere.